src/Infrastructure/Domain/Doctrine/DoctrineSymfonyUser.php

```diff
@@ -9,18 +9,19 @@
 use Becklyn\Security\Domain\Role;
 use Becklyn\Security\Domain\RoleAddedToUser;
 use Becklyn\Security\Domain\RoleRemovedFromUser;
-use Becklyn\Security\Domain\UserCreated;
 use Becklyn\Security\Domain\User;
+use Becklyn\Security\Domain\UserCreated;
 use Becklyn\Security\Domain\UserDisabled;
 use Becklyn\Security\Domain\UserEnabled;
 use Becklyn\Security\Domain\UserId;
 use Becklyn\Security\Infrastructure\Domain\Symfony\SymfonyUser;
 use Doctrine\ORM\Mapping as ORM;
 use Gedmo\Mapping\Annotation as Gedmo;
```

Comment:
Oh, if you have the time, can you please kick out the Gedmo library? That would require going through all doctrine entities, looking for where it is used on $createdTs and $updatedTs props, removing it, and then setting the $createdTs prop manually in the constructor, and the $updatedTs both in the constructor and in any place where the entity is updated, which in this case means in any entity method that raises an event. You could set the same value to $updatedTs as the value which is passed to the event as its $raisedTs param.

Call me if you need more info.